cluster_utils/settings.py

```diff
@@ -222,6 +248,16 @@ def sanitize_numpy_torch(possibly_np_or_tensor):
 
 
 def save_metrics_params(metrics, params):
+    """Send results to the cluster_utils server.
```

Comment:
```suggestion
    """Save metrics and parameters to the working directory and send metrics to the cluster_utils server.
```

(or similar)

Reply:
Out of scope of this PR but looking at this function I realised that the parameters are currently saved twice:  Once to a JSON file settings.json by `read_params_from_cmdline` (if `save_params=True`, which is the default) and then again to a CSV file param_choise.csv by `save_metric_params`.
Is the latter actually used for anything?  If not, I'd propose to make saving the JSON-file non-optional (i.e. remove the argument from `read_params_from_cmdline`) and remove the code for writing param_choise.csv.

Reply:
Well, I think the CSV output is very handy that you can load that into an excel sheat or panda for analysis. You don't want to parse a JSON for that.

Reply:
I see.  Then let's keep it.